etna/datasets/tsdataset.py

```diff
@@ -515,9 +525,17 @@ def target_components_names(self) -> Tuple[str, ...]:
         return self._target_components_names
 
     @property
+    @deprecated(
+        reason="Usage of this property may mislead while accessing prediction intervals. Use `prediction_intervals_names` property to access intervals names!"
```

Comment:
Don't we want to set a version in which it is going to be removed?

Reply:
Idealy, it should be removed when new prediction intervals are completely stable and moved from experimental. Right now, it seems like an open question.

Reply:
We can't remove this property without major update. So, let's set version 3.0 for example for this property to be removed.